scrapy/utils/reqser.py

```diff
@@ -1,95 +1,22 @@
-"""
-Helper functions for serializing (and deserializing) requests.
-"""
-import inspect
+import warnings
+from typing import Optional
 
-from scrapy.http import Request
-from scrapy.utils.python import to_unicode
-from scrapy.utils.misc import load_object
+import scrapy
+from scrapy.exceptions import ScrapyDeprecationWarning
+from scrapy.utils.request import request_from_dict as _from_dict
 
 
-def request_to_dict(request, spider=None):
-    """Convert Request object to a dict.
+warnings.warn(
+    ("Module scrapy.utils.reqser is deprecated, please use scrapy.Request.to_dict"
```

Comment:
```suggestion
    ("Module scrapy.utils.reqser is deprecated, please use request.to_dict method"
```

What do you think about ^^? It is a bit more confusing, but a bit more precise as well; e.g. if you're serializing a custom request, you shouldn't be using scrapy.Request.to_dict.